environment.rb

```diff
@@ -0,0 +1,102 @@
+module Environment
+	class Depot
+		attr_accessor :packs
+		def initialize
+			@packs = {
+				:simple_transportation_pack => {
```

Comment:
When using symbols as keys, the rocket notation is not recommended, instead it would be fine set the Hash elements as:
`simple_transportation_pack: { ... }`